Pipfile

```diff
@@ -10,10 +10,11 @@ apache-airflow-providers-celery = ">=3.0"
 apache-airflow-providers-snowflake = "~=5.2"
 sagetasks = "~=0.4"
 connexion = "==2.14.2"
-py-orca = {version = "==1.3.3", extras = ["all"]}
+py-orca = {version = "==1.3.4", extras = ["all"]}
```

Comment:
I think something might be wrong here - Can you double check that you have main pulled into your branch. The pipfile in main looks like:
```
[packages]
apache-airflow = "==2.7.2"
apache-airflow-providers-amazon = "~=7.2"
apache-airflow-providers-celery = ">=3.0"
apache-airflow-providers-snowflake = "~=5.2"
sagetasks = "~=0.4"
connexion = "==2.14.2"
py-orca = {version = "==1.3.3", extras = ["all"]}
slack-sdk = ">=3.27"
# See https://stackoverflow.com/questions/77846645/airflow-db-migrate-is-giving-typeerror-sqlalchemysessioninterface-init-mi
flask-session = "==0.5.0"
```

Which doesn't match your before/after